### ISSUE TITLE
Update TOC links with REX links 

### ISSUE THREAD

**Issue by allisonhowell** [openstax/business-intel#497]
# Description
REX books are being released starting in September. In phase 1 of the book release, the ToC on the book details page will continue to have links to CNX, and those CNX links will redirect to REX (302 redirects).

*At some point (after the REX, CE, and BIT PDM's agree we are ready to do this)*, we want both the View Online link AND the ToC on the book details page to point straight to REX, so that REX book links gain link authority and so that we aren't sending users from book details -> cnx -> rex. The redirects will work, but it's safer and better for us to point straight to REX once REX is public/stable. 

Release dates are here: https://docs.google.com/spreadsheets/d/1HaaboEhiSAdk-7owNz7QCUkMHO8UfIluZ5AB7TygV00/edit#gid=1406974781

CNX archive includes components of Rex links.
https://archive-staging.cnx.org/contents/4abf04bf-93a0-45c3-9cbc-2cefd46e68cc@11.1.json

# Acceptance Criteria
- When a book is marked as a REX book (I believe this happens manually with that "is rex book" flag in the CMS), the View Online links and the TOC for that REX book point directly to the corresponding pages in REX, not the CNX pages. 
- Books that are not yet in REX continue to have their ToC's and View Online links point to the CNX pages.
- Don't merge/release this until REX and BIT PDM's say we're ready to start having all ToC's for REX books point to REX, because there is an interim period in which REX books' ToC's will still point to CNX.

derive these from "Acceptance Criteria" during "Design" phase

**title**: template  
> GIVEN something  
> AND something else  
> WHEN something  
> AND something else  
> THEN something  
> AND something else  

# Checklist for Done
1A-DEFINE
- [ ] Description is complete 
- [ ] High-level natural language acceptance tests are complete
- [ ] If UX design is needed then the UX label is added
- [ ] If UX design is needed then the UX LoFi Design is completed, approved by the Product Manager, and added to the issue.
- [ ] A high-level solution (development approach for the code change) is added to the issue
- [ ] If prototype code is generated, it is linked to the issue.
- [ ] If tech briefs are generated, they are linked to the issue.

1B-DECOMP
- [ ] The work is broken into reasonable and consistent work items.
- [ ] Story issues are created for the work items, with Description, Acceptance, the label 'story', and an Estimate.
- [ ] The original issue is changed to an epic and the child story issues are linked to it.  The estimate is removed.
- [ ] The story epic and child work item issues are moved into 1B-DECOMP DONE.
- [ ] If an issue is included in a product phase, then Release is selected.

1C-DESIGN
- [ ] The UX HiFi Design is complete, approved by the Product Manager, and added to the issue.
- [ ] Distilled acceptance tests are added to the issue.
- [ ] Automated testing criteria are added to issue.
- [ ] The solution is updated.

2-CODE
- [ ] A pull request is opened, the automated pull request checks pass, and the change has been approved by other developers.
- [ ] The acceptance tests are finalized in "Given / When / Then" format.
- [ ] Regression test categories are identified on the issue. Categories should be referenced if any changes might affect them, even if the intended functionality is unchanged.

4A-UX REVIEW
- [ ] The pull request is merged into master, and the change branch is deleted.

5A-FUNCT VER
- [ ] All the acceptance tests for this issue pass.

5C-REGRESSION
- [ ] The test plan in testrail passes.
- [ ] Blessing from Product Manager that the release is good to go.

5D-READY TO BE DEPLOYED
- [ ] Production is updated and verified.
- [ ] All issues in the release are closed.

**QA needs checklist:**
- [ ] Acceptance Criteria are complete
- [ ] Milestone should be tagged for PR's 
- [ ] Milestone should be tagged for issues
- [ ] Content update needed
- [ ] Content update complete

**Deployment needs**
- [ ] Pre-deployment tasks
- [ ] Data migrations 
- [ ] Post-deployment tasks

_If this issue becomes an epic, delete from 1C-DESIGN to here; otherwise, delete from here to the end_

TRACK / EPICS
- [ ] All child issues are moved to 3B-RELEASE REVIEW.

3B-RELEASE REVIEW
- [ ] Acceptance tests pass for all issues in the release
- [ ] Blessing from Product Manager that the release is good to go.

3C-RELEASED
- [ ] Production is updated and verified.
- [ ] All issues in the release are closed.

**Comment by BruceP99166** [comment]
Michael will set up a meeting to discuss technical architecture.

**Comment by Brooke-Fail** [comment]
Verified in QA, Rex books point to the Rex ToC. Will verify more in Staging.

**Comment by Brooke-Fail** [comment]
Verified in QA.

**Comment by Brooke-Fail** [comment]
Verified in Staging.

**Comment by Brooke-Fail** [comment]
We are not ready to put Rex links in the cms just yet. Verifying based on staging testing, will reopen or create a new card if there are any issues.
